### DISCUSSION TITLE
[archived] "Daemon Containers Walk Through" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#12770] [General]
# en/latest/walk-through/daemon-containers/

https://argo-workflows.readthedocs.io/en/latest/walk-through/daemon-containers/

**Comment by jacek-jablonski** [comment]
Can daemon containers be referenced by name and not by IP?

**Reply by agilgur5** [reply]
That might require creating a `Service`. It's not a sidecar, so you wouldn't be able to use `localhost` through the container runtime.
